data-raw/SLS.R

```diff
@@ -167,16 +182,16 @@ SLS <- waterInfo %>%
   full_join(towInfo,
             by = c("Date", "Station")) %>%
   full_join(catch,
-            by = c("Date", "Station", "Tow")) %>%
+            by = c("Date", "Station", "Tow"),multiple="all") %>%
   full_join(lengths,
-            by = c("Date", "Station", "Tow", "FishCode")) %>%
+            by = c("Date", "Station", "Tow", "FishCode"),multiple="all") %>%
   # Adding in taxa name based on Species Code.csv file
   left_join(Species %>%
               select(TMM_Code,
                      Taxa) %>%
-              filter(!is.na(TMM_Code)),
+              dplyr::filter(!is.na(TMM_Code)),
             by = c("FishCode"="TMM_Code")) %>%
-  left_join(SLSTables$`20mm Stations`,
+  left_join(SLSTables$"20mm Stations",
```

Comment:
Does this work? I thought you had to use backticks after a $ instead of quotes?

Reply:
Apparently it does. I agree it looks really weird though. Regardless, removed this as this table is now replaced with `SLS Stations` in the SLS Access db. Updated the code accordingly